app/uk/gov/hmrc/plasticpackagingtax/registration/views/components/errorSummary.scala.html

```diff
@@ -22,11 +22,15 @@
 
 @errorList = @{errors.map { error =>
     val errorKey = error.key.replace(".", "_").replace('[', '_').replace("]", "")
```

Comment:
what is this :S

Reply:
this should be the ID of the input field 🤔

Reply:
if we just removed this replacing logic does it work naturally?

Reply:
it does for some of the error links on date but not all - the fix above sends all error links for that field to the date day input